packages/settingeditor/src/plugineditor.ts

```diff
@@ -18,7 +18,7 @@ import { Message } from '@lumino/messaging';
 import { ISignal, Signal } from '@lumino/signaling';
 import { StackedLayout, Widget } from '@lumino/widgets';
 import { RawEditor } from './raweditor';
-import { SettingEditor } from './settingeditor';
+import { SettingEditor } from './jsonsettingeditor';
```

Comment:
It would be good to rename `SettingEditor` to `JSONSettingEditor` to make things more obvious